sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuHashJoin.scala

```diff
@@ -256,9 +256,35 @@ object GpuHashJoin {
   }
 }
 
+/**
+ * Class to hold statistics on the build-side batch of a hash join.
+ * @param streamMagnificationFactor estimated magnification of a stream batch during join
+ * @param isDistinct true if all build-side join keys are distinct
+ */
+case class JoinBuildSideStats(streamMagnificationFactor: Double, isDistinct: Boolean)
+
+object JoinBuildSideStats {
+  // TODO: Make boundBuildKeys Seq[GpuExpression]
```

Comment:
Is this TODO needed?

Reply:
Ah, I meant to fix this up and forgot.  It makes the API more type-safe, so I'd like to clean that up.